onnxruntime/core/providers/coreml/builders/impl/activation_op_builder.cc

```diff
@@ -95,6 +95,19 @@ Status ActivationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
   } else if (op_type == "PRelu") {
     auto* prelu = layer->mutable_activation()->mutable_prelu();
     ORT_RETURN_IF_ERROR(AddPReluWeight(model_builder, node, logger, *prelu));
+  } else if (op_type == "LeakyRelu") {
+    const auto& attributes = node.GetAttributes();
+    auto alpha_attr = attributes.find("alpha");
+
+    if (alpha_attr != attributes.end()) {
+      auto alpha = (*alpha_attr).second.f();
+      auto *leaky_relu = layer->mutable_activation()->mutable_leakyrelu();
```

Comment:
you can use the NodeAttributeHelper to get the `alpha` attribute, as what we do in other places, for example:

https://github.com/microsoft/onnxruntime/blob/786f8b98f7a8ed6165212bed48c7c01fc86ea624/onnxruntime/core/providers/coreml/builders/impl/LRN_op_builder.cc#L41-L43